main.tf

```
@@ -35,24 +33,45 @@ locals {
     label_order         = var.label_order == null ? var.context.label_order : var.label_order
     regex_replace_chars = var.regex_replace_chars == null ? var.context.regex_replace_chars : var.regex_replace_chars
     id_length_limit     = var.id_length_limit == null ? var.context.id_length_limit : var.id_length_limit
+    label_value_case    = var.label_value_case == null ? var.context.label_value_case : var.label_value_case
+    label_key_case      = var.label_key_case == null ? var.context.label_key_case : var.label_key_case
```

It's a nitpick, and I wouldn't complain about this if I weren't also suggesting more substantive changes
```suggestion
    label_key_case      = var.label_key_case == null ? var.context.label_key_case : var.label_key_case
    label_value_case    = var.label_value_case == null ? var.context.label_value_case : var.label_value_case
```